std_msgs/Header header
geometry_msgs/Point[] blue_cones
geometry_msgs/Point[] large_orange_cones
geometry_msgs/Point[] orange_cones
geometry_msgs/Point[] unknown_cones
geometry_msgs/Point[] yellow_cones
